src/main.rs

```diff
@@ -19,19 +19,21 @@ fn main() {
     let token = get_token();
     let epic = random_msg();
 
-    if let Err(token) = token {
-        println!("Couldn't post tweet ({}), reason: Failed to find an .env variable: {:?}", epic, token);
-    } else {
-        
-        let tweet = DraftTweet::new(&epic);
-        
-        let mut core = Core::new().unwrap();
-        let handle = core.handle();
-        let result = core.run(tweet.send(&token.unwrap(), &handle));
-        match result {
-            Ok(_) => println!("Successfully posted tweet: {}", epic),
-            Err(err) => println!("Couldn't post tweet ({}), reason: {}", epic, err),
-        }
+    match token {
+        Err(token) => println!("Couldn't post tweet ({}),
+            reason: Failed to find an .env variable: {:?}", epic, token),
```

Comment:
This string is going to contain all this indentation white space, is that what you want?
Also, I think parenthesizing a multi-line string isn't the greatest way to handle it.
Maybe this error should be more like:

```
Failed finding environment variable: {:?}

{}
```